shared/src/main/scala/squants/energy/Energy.scala

```diff
@@ -187,6 +198,46 @@ object Ergs extends EnergyUnit {
   val symbol = "erg"
 }
 
+object ElectronVolt extends EnergyUnit {
+  val conversionFactor = Joules.conversionFactor * 1.60217656535e-19
```

Comment:
Thanks for poiting this up, I'll update this and the constants PR that uses the same bad logic